breathecode/media/views.py

```diff
@@ -420,3 +424,33 @@ def get(self, request, media_id=None, media_slug=None):
             resource[header] = response.headers[header]
 
         return resource
+
+class ResolutionView(APIView):
+    @capable_of('read_media')
+    def get(self, request, media_id=None, academy_id=None):
+
+        if media_id:
+            items = MediaResolution.objects.filter(media_id=media_id)
+
+            if not items:
+                raise ValidationException('Resolutions for this media were not found', code=404, slug='media-resolutions-not-found')
+
+        items = MediaResolution.objects.filter(media_id=media_id)
+        sort = request.GET.get('sort')
+        if sort:
+            items = items.order_by(sort)
+
+        serializer = GetResolutionSerializer(items, many=True)
```

Comment:
In the `MaskingUrlView`, check if exists the request resolution in other academy, if exist we just perform a copy of this resolution for this academy without create a new copy of this resolution in google cloud storage